### PR TITLE
Request-spec

### PR DESCRIPTION
**Pull request by Yosefgeda**
- Added RSpec as a gem.
- Created a request spec.
- tests for:
1. If the response status was correct.
2. If a correct template was rendered.
4. If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
```
Comment (rustam-tolipov) [inline spec/request_spec.rb]: You did great job!
I can see your user request tests are working. However, we need a little more ajjustements for it :)

- [ ] For starting, please create a new folder that called `request` and put your request tests in it.

File: spec/request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+describe UsersController, type: :controller do
```
Comment (rustam-tolipov) [inline spec/request_spec.rb]: - [ ] Second, please create Post requests tests as well and it should cover `index` and `show` methods of controller.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Yosefgeda,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well implemented code_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Yosefgeda, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Good job with test coverage

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**